packages/ui-extensions/src/surfaces/admin/components/DatePicker/DatePicker.doc.ts

```diff
@@ -0,0 +1,43 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'DatePicker',
+  description:
+    'Date pickers let merchants choose dates from a visual calendar that’s consistently applied wherever dates need to be selected across Shopify.    ',
+  requires: '',
+  thumbnail: 'datepicker-thumbnail.png',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'DatePickerProps',
+      description: '',
+      type: 'DatePickerProps',
+    },
+  ],
+  category: 'Components',
+  subCategory: 'Forms',
+  defaultExample: {
+    image: 'datepicker-default.png',
```

Comment:
I don't really know if we need this image in _this_ repo, but the other components have it

Reply:
We should be consistent about providing images for all components that have UI. This is not optional IMO.